client/helpers.go

```diff
@@ -268,3 +269,59 @@ func serializeCanonicalRole(tufRepo *tuf.Repo, role data.RoleName, extraSigningK
 
 	return json.Marshal(s)
 }
+
+func isMetaCached(cache store.MetadataStore) (bool, error) {
+	if cache == nil {
+		return false, nil
+	}
+
+	for _, role := range data.BaseRoles {
+		_, err := cache.GetSized(role.String(), store.NoSizeLimit)
+		if err != nil {
+			if _, ok := err.(store.ErrMetaNotFound); ok {
+				continue
```

Comment:
@n4ss - sorry yes, you're correct.  Could we add a docstring-style comment to this function?